### ISSUE TITLE
Allow notes to be published and shared with anyone who wants to read them

### ISSUE THREAD

**Issue by scottbenton** [scottbenton/Iron-Journal#402]

**Comment by nickarrow-temp** [comment]
Copy and paste works pretty well for this today. I could see a future where a note could be “exported” or “copied to the clipboard” for sharing/blogposting/whatever. 

“Copy note to clipboard” might be the easiest to implement, knowing that way out in your future/backlog is #156

**Comment by scottbenton** [comment]
You're correct, and you can copy/paste from a note today. This is more so for allowing people to share their stories with others. So that publishing and sharing something like this can be possible from within Crew Link.

https://zkat.tech/campaigns/a-lantern-in-the-void/chapter-07-beyond-beryl/
